libs/ui-watt/src/lib/components/input/datepicker/watt-date-range-chip.component.ts

```diff
@@ -67,32 +72,28 @@ export class EndOfDaySelectionStrategy extends DefaultMatCalendarRangeStrategy<D
       [opened]="picker.opened"
       (toggle)="picker.open()"
     >
-      <mat-date-range-input #input class="cdk-visually-hidden" separator="―" [rangePicker]="picker">
-        <input
-          type="text"
-          matStartDate
-          tabindex="-1"
-          [value]="value?.start"
-          (dateChange)="value = input.value ?? undefined"
-          (dateChange)="selectionChange.emit(input.value ?? undefined)"
-        />
+      <mat-date-range-input #input class="cdk-visually-hidden" separator="" [rangePicker]="picker">
+        <input type="text" matStartDate tabindex="-1" role="none" [value]="value?.start" />
```

Comment:
issue: maybe removing `dateChange` from this input causes a weird issue

When I try to select a date after another date is already selected, the first click doesn't do anything. A second click is necessary to select the "from" date and a third click for the "to" date.

![issue](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/e6be805d-9efb-4699-9bf6-c7847aaca54c)

Reply:
https://github.com/Energinet-DataHub/greenforce-frontend/pull/1804/commits/fe51287abfa90d890b641f9358b71f78bfcd39cd